Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -43,52 +47,47 @@ public float groundedMoveSpeed
 	[SerializeField] protected float jumpMoveSpeed = 6f;
 	[SerializeField] protected float rollMoveSpeed = 6f;
 	[SerializeField] protected float climbMoveSpeed = 6f;
+	#endregion
 
-	// Model Info
-	[Space(10)][Header("Model Info")]
+	#region Collisions
+	[Space( 10 ), Header( "Collisions" )]
+	public SphereCollider bumper;
+	private Transform _bumperTransform;
+	private Rigidbody _bumperRigidbody;
+	#endregion
+
+	#region Model Info
+	[Space( 10 ), Header( "Model Info" )]
 	public Transform model;
 	[SerializeField] float modelTurnSpeed = 7f;
-	protected Vector3 modelOffset;
-
-	// Slope Checking
-	[Space(10)][Header("Slope Checking")]
-	[SerializeField] float groundSlopeCheckRadius = 0.2f;
-	[SerializeField] float groundSlopeCheckHeight = 0.7f;
-	float groundSlopeCapsuleWidth = 0.4f;
-	float groundSlopeCastDist = 0.3f;
-
-	float groundSlopeFallSpeed = -30f;
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float lookDirLerp = 0.002f;
 
-	protected float groundSlopeSpeedMod = 1f;
-	float groundSlopeSpeedModeDelta = 2.5f;
+	private Vector3 _lastPos;
```

Comment:
Why are we labeling private variable with underscores? Looks weird and doesn't really give any extra info. If a variable is private, I'll notice when I try to use it.